### DISCUSSION TITLE
ERPNext and Frepple integration

### DISCUSSION THREAD

**Discussion by Drayang** [frePPLe/frepple#501] [Show and tell]
Recently i was working on a ERP system called ERPNext and was trying to  integrate Frepple with ERPNext in order to utilise the Frepple as the production scheduling tool. I successfully developed a custom app (i.e a common concept in ERPNext to add on feature without changing the original source code) that act as a connector to transfer data between frepple and ERPNext. The idea is by using the REST API to have a bidirectional data transfer. Besides, I used the data mapping for Odoo connector as the reference and end up having a pretty similar data mapping for ERPNext and Frepple. Here is the github link for the app i have created: [msf4-0/ERPnext- Frepple-Integration](https://github.com/msf4-0/ERPNext-Frepple-Integration.git) and I wonder whether you are interested to see it as one of the potential connector or feature that can be added to the Frepple Production Scheduling software?

Thank you!

**Comment by jdetaeye** [comment]
Interesting & cool!  We welcome such integrations from the frepple community.

To promote your integration we'll create some links to your github repository from the frepple documentation.

**Comment by FelipeTesta** [comment]
Really interesting, I was studying the ERPNext system all of the past month and just today I fall in this post. Really great ideia, bc by my search, this software is really needed in a factory PCP. I'm still learning, but sure I'll be using both of them in the future and this comunication will be promissing.
